controllers/validator.go

```diff
@@ -78,6 +78,16 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		}
 	}
 
+	// check for VSM/Volsync DataMover (OADP 1.2 or below) syntax
+	for _, plugin := range dpa.Spec.Configuration.Velero.DefaultPlugins {
+		if plugin == oadpv1alpha1.DefaultPluginVSM {
+			return false, errors.New("Delete vsm from spec.configuration.velero.defaultPlugins and dataMover object from spec.features. Use Velero Built-in Data Mover instead")
+		}
+	}
```

Comment:
Move this into [`func (r *DPAReconciler) ValidateVeleroPlugins(log logr.Logger) (bool, error) {`](https://github.com/openshift/oadp-operator/blob/db116623f3e66ad9763514d562b688624cc44d00/controllers/validator.go#L121) ?

Reply:
there's already a for loop there.

Reply:
going to check that out

Reply:
done